.travis.yml

```diff
@@ -19,7 +19,7 @@ env:
       NUMPY_VERSION="1.8.2" SCIPY_VERSION="0.13.3"
       SCIKIT_LEARN_VERSION="0.14.1"
       PANDAS_VERSION="0.13.0"
-    - DISTRIB="neurodebian" PYTHON_VERSION="2.7"
+    # - DISTRIB="neurodebian" PYTHON_VERSION="2.7"
```

Comment:
This is probably fine as a temporary hack, but you probably want to understand the error at some point because neurodebian is important for the target audience ...